src/main/webapp/recommendation-map.html

```diff
@@ -29,7 +29,8 @@ <h3>a recommendation system for students, by students!</h3>
         </ul>
       </nav>
     </div>    
-    <button onclick="toTopFunction()" id="to-top-button">Up to Top</button>
+    <!-- TODO debug this as it causes a console error 
+    <button onclick="toTopFunction()" id="to-top-button">Up to Top</button> -->
```

Comment:
I added a comment in `common.js` which probably should resolve this error. Please give it a try and let me know if it does :)

Reply:
Thank you so much! It worked :)